.circleci/config.yml

```diff
@@ -22,68 +21,84 @@ restore_node_modules: &restore_node_modules
 
 
 # Main Config
-version: 2
-jobs:
-  install:
-    <<: *job_common
-    steps:
-      - checkout
-      - run: echo "//registry.npmjs.org/:_authToken=$NPM_TOKEN" >> .npmrc
-      - run:
-          name: Update NPM and Install dependencies
-          command: |
-            if [ ! -d node_modules ]; then
-              sudo npm install -g npm@6
-              npm ci
-            fi
-      - run:
-          name: Copy node_modules to workspace
-          command: mv node_modules /tmp/
-      - persist_to_workspace:
-          root: /tmp
-          paths:
-            - node_modules
-      - run:
-          name: Save NPM install log
-          command: npm ls --json > /tmp/npm_install.log || true
-      - store_artifacts:
-          path: /tmp/npm_install.log
-          destination: npm-install
-
-  static_analysis:
-    <<: *job_common
-    steps:
-      - checkout
-      - <<: *restore_workspace
-      - <<: *restore_node_modules
-      - run:
-          name: Run linting
-          command: npm run lint:ci -- --quiet
-      - store_test_results:
-          path: reports/style
-      - store_artifacts:
-          path: reports/style
-          destination: style
-
-  build:
-    <<: *job_common
-    environment:
-      - NODE_ENV: "test"
-    steps:
-      - checkout
-      - <<: *restore_workspace
-      - <<: *restore_node_modules
-      - run:
-          name: Build all packages
-          command: npm run build
-      - persist_to_workspace:
-          root: ./
-          paths:
-            - packages/node_modules
+install:
+  <<: *job_common
+  steps:
+    - checkout
+    - run: echo "//registry.npmjs.org/:_authToken=$NPM_TOKEN" >> .npmrc
+    - run:
+        name: Update NPM and Install dependencies
+        command: |
+          if [ ! -d node_modules ]; then
+            sudo npm install -g npm@6
+            npm ci
+          fi
+    - run:
+        name: Copy node_modules to workspace
+        command: mv node_modules /tmp/
+    - persist_to_workspace:
+        root: /tmp
+        paths:
+          - node_modules
+    - run:
+        name: Save NPM install log
+        command: npm ls --json > /tmp/npm_install.log || true
+    - store_artifacts:
+        path: /tmp/npm_install.log
+        destination: npm-install
+static_analysis:
+  <<: *job_common
+  steps:
+    - checkout
+    - <<: *restore_workspace
+    - <<: *restore_node_modules
+    - run:
+        name: Run linting
+        command: npm run lint:ci -- --quiet
```

Comment:
We should probably add `--quiet` to the `lint:ci` command by default.

Reply:
Might be easier to see all the output by just copying the command and removing the quiet flag manually